### ISSUE TITLE
Command on page /preamble/how-to-use.html to create the conda environment no longer works?

### ISSUE THREAD

**Issue by JackScheff** [ProjectPythia/pythia-foundations#485]
On https://foundations.projectpythia.org/preamble/how-to-use.html under Interacting with Jupyter Books Locally, the recommended command to create Pythia's conda environment does not seem to execute.  The command is

**conda env create --force -f https://raw.githubusercontent.com/ProjectPythia/pythia-foundations/main/environment.yml**

However, on running it (after having installed and initialized the latest version of Conda using miniconda), it returns:

**conda: error: unrecognized arguments: --force**

Is this argument no longer supported in the current version of Conda?  Does the recommended command on this page therefore need to be updated?  Or am I missing something?  I am new to Conda, so the latter may be the case.

**Comment by brian-rose** [comment]
Hi @JackScheff, thanks for this report!

Looks like you are right, the `--force` option was removed in a [conda release earlier this year](https://github.com/conda/conda/releases/tag/24.3.0).

The new syntax for the same functionality is to replace `--force` with `--yes`, so the full command should be 
```
conda env create --yes -f https://raw.githubusercontent.com/ProjectPythia/pythia-foundations/main/environment.yml
```

Or, you can omit the `--yes` entirely -- but then you'll have to click `Y` to manually accept the proposed packages after conda finished solving the environment.

We'll get those docs updated.

**Comment by JackScheff** [comment]
Thank you so much Brian - this worked great!  I'm new to all of this but
quickly learning.  -Jack

**Comment by brian-rose** [comment]
Fixed in #487. 

The new instructions are now published at https://foundations.projectpythia.org/preamble/how-to-use.html#interacting-with-jupyter-books-locally.
